docs/design/lexical_conventions/words.md

```diff
@@ -29,6 +29,10 @@ the precise rules are decided; see the
 [Unicode source files](/proposals/p0142.md#characters-in-identifiers-and-whitespace)
 proposal.
 
+The Carbon source files (including comments and string literals) are required to
+be in Unicode Normalization Form C (NFC). The Carbon source formatting tool will
+convert source files to NFC as necessary in order to satisfy the constraints.
```

Comment:
I think we should drop this sentence, since it's more about the source formatting tool than the language itself.

```suggestion
be in Unicode Normalization Form C (NFC).
```

Reply:
Hmm, you are right!